wayland-backend/src/client_api.rs

```diff
@@ -146,28 +178,20 @@ impl Backend {
 
     /// Get the last error that occurred on this backend
     ///
-    /// If this returns an error, your Wayland connection is already dead.
```

Comment:
Rustdoc requires you surround the type like [`Some`] for doc links to work

Reply:
I didn't really plan to put a link here, you think it's needed?

Reply:
Not strictly needed, but I'd personally have it become a link